docs/sources/architecture/configuration.md

```diff
@@ -29,6 +29,13 @@ Configuration descriptions must be placed as XML file(s) (with the ending `.xml`
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
   <config-description uri="{binding|thing-type|bridge-type|channel-type|any_other}://bindingID:...">
+    <group name="String">
```

Comment:
No, it´s the schema. We defined sequences, which has a fixed order as far as I know.